app/institutions/index/template.hbs

```diff
@@ -19,7 +19,7 @@
                             data-test-read-more-link
                             data-analytics-name='Read more'
                             local-class='Institutions__block__link'
-                            @href='https://cos.io/our-products/osf-institutions/'
+                            @href='https://www.cos.io/institutions/'
```

Comment:
Definitely a nit, but do we need the `www` in the link?

Reply:
it's not necessary, and within EOW, it looks like we just use `https://cos.io/` so I'll strip the `www` :+1: